cmd/start.go

```diff
@@ -42,7 +45,14 @@ func startNetwork(cmd *cobra.Command, args []string) error {
 	ctx := binutils.GetAsyncContext()
 
 	ux.Logger.PrintToUser(startMsg)
-	_, err = cli.LoadSnapshot(ctx, snapshotName)
+
+	outputDirPrefix := path.Join(app.GetRunDir(), "restart")
```

Comment:
shall also use the current rundir of the server? as the deploy

Reply:
since it's restarting, it gets a new folder